net-irc/bitchx/bitchx-1.2.1.recipe

```diff
@@ -12,7 +12,7 @@ Features:
 * Huge installed user base, an abundance of scripts, and a dedicated help \
 channel (#BitchX on EFNET).
 * Built-in screen client, allowing you to detach and re-attach to an irc session."
-HOMEPAGE="http://www.bitchx.com/"
+HOMEPAGE="http://www.bitchx.org"
```

Comment:
Suggestion: add a trailing slash to [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage)
```diff
-HOMEPAGE="http://www.bitchx.org"
+HOMEPAGE="http://www.bitchx.org/"
```